src/index.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
-var bind = require('function-bind');
+var hasOwnProperty = {}.hasOwnProperty;
+var call = hasOwnProperty.call;
```

Comment:
```suggestion
var call = Function.prototype.call;
```
better to minimize indirection.

Reply:
It's for better minification. However, it's not principal.